Move the Linux service to the main block

pkgw opened the pull request:
It wants to access Redis, etc., so it should all go together.

Also comment out the "datatier" storage, which isn't used in production. (There's probably a better variable-ier way to do this.)

I wanted to rename the current "wwt" service/plan to "communities" to reflect its reduced role, but Terraform would want to destroy and recreate the resources in order to do so, which I don't want to do even though the whole point of Terraform is that it's supposed to be safe ...

pkgw committed:
website: comment out storage account that's unused in prod

pkgw committed:
Import new .terraform.lock.hcl files

pkgw committed:
Update the .NET5 app service to run on a Linux plan with our new data Docker image

pkgw committed:
coredata: remove, merged into "website"

pkgw committed:
website: update outputs and comment on nomenclature

pkgw committed:
website: sync up some minor settings with ground truth

coderandhiker commented on website/.terraform.lock.hcl:
```diff
@@ -0,0 +1,19 @@
+# This file is maintained automatically by "terraform init".
```
@pkgw this should be excluded from your commit - add a record to your .gitignore for everything under .terraform and the .terraform.lock.hcl if the aren't in there already

pkgw replied:
@coderandhiker The Terraform CLI specifically instructs that these lockfiles should be kept in version control. I think it's a more recent thing, it only appeared when I upgraded my OS a week or two ago.

coderandhiker replied:
Ok, that's new to me - go ahead and add it back in. I found the ref: https://discuss.hashicorp.com/t/terraform-0-14-the-dependency-lock-file/15696

coderandhiker reviewed (changes requested):
We cannot dual deploy Windows and Linux App Service Plans into the same Resource Group at this time.  This is on the roadmap: https://feedback.azure.com/forums/169385-web-apps/suggestions/37287583-allow-a-linux-and-windows-app-service-plan-to-exis

I recommend adding a new var.prefix-linux-resources resource group and updating the app service plan and app service to deploy into it.

pkgw committed:
website: need to create a separate Linux resource group

Cf: https://feedback.azure.com/forums/169385-web-apps/suggestions/37287583-allow-a-linux-and-windows-app-service-plan-to-exis

coderandhiker reviewed (approved):
I ran the revisions in my subscription and everything looks good now.
